docs/zh_cn/tutorials/How_to_support_new_model.md

```diff
@@ -0,0 +1,86 @@
+## 如何支持新的模型
```

Comment:
Better to change filename  from `How_to_support_new_model.md` to `how_to_support_new_models.md`, which is aligned with the English file.